From 8246b2182b9e9ff0497283bcdb1068d5dc2df6c6 Mon Sep 17 00:00:00 2001
From: Sam Yang <r52096@freescale.com>
Date: Tue, 10 Jun 2008 17:07:19 +0800
Subject: [PATCH] ENGR00069395 gui compiling error by true false definition

Resolved gui compiling error caused by true/false definition in header file.

Signed-off-by: Sam Yang <r52096@freescale.com>
---
 include/asm-arm/arch-mxc/pmic_external.h |    8 --------
 1 files changed, 0 insertions(+), 8 deletions(-)

diff --git a/include/asm-arm/arch-mxc/pmic_external.h b/include/asm-arm/arch-mxc/pmic_external.h
index 9284f4f..518d30f 100644
--- a/include/asm-arm/arch-mxc/pmic_external.h
+++ b/include/asm-arm/arch-mxc/pmic_external.h
@@ -36,14 +36,6 @@
 #include <asm/ioctl.h>
 #include <asm/arch/pmic_status.h>
 
-#ifndef __KERNEL__
-typedef _Bool bool;
-enum {
-	false = 0,
-	true = 1
-};
-#endif
-
 /*!
  * This is the enumeration of versions of PMIC
  */
-- 
1.5.4.4

